srcpkgs/Solaar/template

```diff
@@ -1,15 +1,20 @@
 # Template file for 'Solaar'
 pkgname=Solaar
-version=1.0.7
+version=1.1.4
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-pyudev python3-psutil python3-yaml python3-xlib"
+depends="python3-evdev python3-pyudev python3-psutil python3-yaml python3-xlib"
 short_desc="Linux device manager for a wide range of Logitech devices"
 maintainer="Young Jin Park <youngjinpark20@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://pwr-solaar.github.io/Solaar/"
+changelog="https://raw.githubusercontent.com/pwr-Solaar/Solaar/master/ChangeLog.md"
 distfiles="https://github.com/pwr-Solaar/Solaar/archive/${version}.tar.gz"
-checksum=39c025b4186b6cb4620bc52d1d20e2d841082982c8be0fed155398faee7a9cd1
+checksum=4a6745d1a7cd78873980ce39fec8e44ecb3e5447875335a0237e1fa755217984
 # Package provides no tests
 make_check=no
+
+post_install() {
+	vdoc README.md
```

Comment:
this readme is the same content as the homepage, and references other pages on that website. Also, most of the useful info seems to be on those other pages. Do you think it's worth it to include?

Reply:
yep you are right. removed it.